docs/en/reference/schema-manager.rst

```diff
@@ -233,6 +233,29 @@ table:
     )
     */
 
+createComparator()
+------------------
+
+To create a comparator that can be used to compare two schemas use the
+``createComparator()`` method which returns an instance of
+``Doctrine\DBAL\Schema\Comparator``.
+
+.. code-block:: php
+
+    <?php
+    $comparator = $sm->createComparator();
+    $schemaDiff = $comparator->compareSchemas($fromSchema, $toSchema);
+
+To change the configuration of the comparator, you can pass as
```

Comment:
```suggestion
To change the configuration of the comparator, you can pass a
```

Reply:
Done in 92356cff13ff6ced66a217c2e69a9811978ec357